packages/mdc-textfield/foundation.js

```diff
@@ -204,7 +207,11 @@ class MDCTextFieldFoundation extends MDCFoundation {
     const labelScale = isDense ? numbers.DENSE_LABEL_SCALE : numbers.LABEL_SCALE;
     const labelWidth = this.adapter_.getLabelWidth() * labelScale;
     const isRtl = this.adapter_.isRtl();
```

Comment:
calculate all this labelWidth and isRTL within the if statement for opening the notch.